CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.12)
+cmake_minimum_required(VERSION 3.15)
```

Comment:
How do you use CMAKE_MSVC_RUNTIME_LIBRARY? I'm not against increasing, but it's strange to increase it for something that is not used anywhere in the build files.

Reply:
It will be needed for this issue: [Pre-built release binaries? #379](https://github.com/skvadrik/re2c/issues/379)
Currently it is used in my repo that produces a Windows binary: [re2c-for-Windows/build.ps1](https://github.com/PolarGoose/re2c-for-Windows/blob/main/build.ps1#L51)